entrypoint.sh

```diff
@@ -6,7 +6,9 @@ python manage.py collectstatic --no-input
 echo "Applying migrations"
 python manage.py migrate
 
-echo "Starting the server and bot..."
+echo "Starting the server, celery and bot..."
 exec "$@"
 
+celery -A hackathon_bot worker --loglevel=info &  celery -A hackathon_bot beat --loglevel=info &
```

Comment:
Celery doesn't start as expected. Here's what I get while trying to build the project 
```
backend-1  | Error: Invalid value for '-A' / '--app': 
backend-1  | Unable to load celery application.
backend-1  | The module hackathon_bot was not found.
backend-1  | Usage: celery [OPTIONS] COMMAND [ARGS]...
backend-1  | Try 'celery --help' for help.
```

Reply:
Resolved and ready for merge.
Functionality tested.